doc/source/user_guide/ug_case_settings.rst

```diff
@@ -338,6 +338,7 @@ Table of namelist options
    "","``restore_ice``", "true/false", "restore ice state along lateral boundaries", ""
    "\*","``atm_data_type``", "``default``", "constant values defined in the code", ""
    "","", "``LYq``", "AOMIP/Large-Yeager forcing data", ""
+   "","", "``JRA55``", "JRA55 forcing data", ""
```

Comment:
Let's add the citation here. You want to do it with the following format :cite:`Tsujino18` (see below for box2001 as an example). 

You will also have to add the citation at the very en of the following file (right after Roberts18: /doc/source/master_list.bib

@Article{Tsujino18,
  author = "H. Tsujino and S. Urakawa and R.J. Small and W.M. Kim and S.G. Yeager and et al.",
  title = "{JRA‐55 based surface dataset for driving ocean–sea‐ice models (JRA55‐do)}",
  journal = OM,
  year = {2018},
  url = {http://dx.doi.org/10.1016/j.ocemod.2018.07.002}
}

Reply:
I added the citation.